### PR TITLE
ls: Refactored options and other long constants to fix formatting

### PR DESCRIPTION
**Pull request by hbina**
Only refactors the `options` module and a bunch of long strings as opposed to full refactor here https://github.com/uutils/coreutils/pull/2434

Signed-off-by: Hanif Bin Ariffin <hanif.ariffin.4326@gmail.com>

### CODE REVIEW

File: src/uu/ls/src/ls.rs [hunk 1]
```diff
@@ -709,84 +649,61 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
...
-
         // Hide and ignore
         .arg(
             Arg::with_name(options::HIDE)
                 .long(options::HIDE)
                 .takes_value(true)
                 .multiple(true)
                 .value_name("PATTERN")
-                .help("do not list implied entries matching shell PATTERN (overridden by -a or -A)")
+                .help(options::HIDE_HELP),
```
Comment (sylvestre) [inline src/uu/ls/src/ls.rs:706]: I am not sure to understand why move this to `options` and not others?
BTW, I am not fan of moving the help somewhere else. it is useful to have the help to know what is is doing.
So, could you please revert this change?

File: src/uu/ls/src/ls.rs [hunk 2]
```diff
@@ -639,41 +572,48 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
...
                 .help("List one file per line.")
-                .multiple(true)
+                .multiple(true),
         )
         .arg(
             Arg::with_name(options::format::LONG_NO_GROUP)
                 .short(options::format::LONG_NO_GROUP)
-                .help("Long format without group information. Identical to --format=long with --no-group.")
-                .multiple(true)
+                .help(options::format::LONG_NO_GROUP_HELP)
```
Comment (sylvestre) [inline src/uu/ls/src/ls.rs:580]: same here

File: src/uu/ls/src/ls.rs (hunk 1 above)
Comment (hbina) [reply src/uu/ls/src/ls.rs:706]: TBH I just tried moving some stuff out and see if it works. I will remove this and see if it works.

File: src/uu/ls/src/ls.rs (hunk 2 above)
Comment (hbina) [reply src/uu/ls/src/ls.rs:580]: Putting the text here will make `cargo fmt` not work. So I am keeping it here.

Comment (sylvestre) [reply src/uu/ls/src/ls.rs:580]: splitting it on several lines will probably fix it

Comment (hbina) [reply src/uu/ls/src/ls.rs]: Nice, I have reduced the fix to just formatting changes with the method you suggested.

**Comment by hbina** [comment]
Any idea how do I see the full report for this run here? https://github.com/uutils/coreutils/pull/2436/checks?check_run_id=2864586345

I can't expand the error ones.

**Comment by sylvestre** [comment]
I can't see it either. I re-triggered the run

**Comment by sylvestre** [comment]
could you please fix the conflict? thanks

**Comment by sylvestre** [comment]
sorry but it is still conflicting

**Comment by hbina** [comment]
Rebased again.

File: src/uu/ls/src/ls.rs [hunk 3]
```diff
@@ -45,19 +45,12 @@ use unicode_width::UnicodeWidthStr;
...
 use uucore::libc::{S_IXGRP, S_IXOTH, S_IXUSR};
 
-static ABOUT: &str = "
- By default, ls will list the files and contents of any directories on
- the command line, expect that it will ignore files and directories
- whose names start with '.'
-";
-static AFTER_HELP: &str = "The TIME_STYLE argument can be full-iso, long-iso, iso.
-Also the TIME_STYLE environment variable sets the default style to use.";
-
```
Comment (rivy) [inline src/uu/ls/src/ls.rs:55]: Why are you moving these?

The current form follows the pattern of other utils, so it should remain unless you have some reason to move them.

**Review by rivy — CHANGES_REQUESTED** [review]

**Comment by rivy** [comment]
Technical nit... use present case for the commit message please.

File: src/uu/ls/src/ls.rs (hunk 3 above)
Comment (ycd) [reply src/uu/ls/src/ls.rs:55]: In addition to @rivy's comment, we can consider moving those constants under our `mod options` but yeah please don't break the consistency for now. 

```rust
pub mod options {
    pub mod help {
        pub static HELP: &str = "help";
    }
}
